custom/conf/app.example.ini

```diff
@@ -613,6 +613,33 @@ WHITELISTED_URIS =
 ; Example value: loadaverage.org/badguy stackexchange.com/.*spammer
 BLACKLISTED_URIS =
 
+[oauth2_client]
+; Whether a new auto registered oauth2 user needs to confirm their email.
+; Do not include to use the REGISTER_EMAIL_CONFIRM setting from the `[service]` section.
+REGISTER_EMAIL_CONFIRM =
+; Scopes for the openid connect oauth2 provider (seperated by space, the openid scope is implicitly added).
+; Typical values are profile and email.
+; For more information about the possible values see https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims
+OPENID_CONNECT_SCOPES =
+; Scopes for the google oauth2 provider (seperated by space)
+; Default scopes will provide only email, but for registration we may need more: email, profile
+GOOGLE_SCOPES =
```

Comment:
do we really need to customize these? Maybe just always request "email profile" if `ENABLE_AUTO_REGISTRATION` is enabled?

Reply:
Profile scope may be usefull even without auto linking. This patch also provides possibility for automatic avatar update from oauth provider.